frontend/views/components/CreateGroupSteps/GroupSummary.vue

```diff
@@ -1,4 +1,5 @@
 <template lang="pug">
+//- TODO: Remove?
```

Comment:
I would say yes. What do you think @taoeffect ?

Reply:
Let's keep it around for now just in case

Reply:
Hum... "if just in case" we need it again we can go back in history (that's why tools like git exist!) :D

Reply:
@sandrina-p then feel free to move it to `/historical` folder?

Reply:
I'll ask an obvious question.. what is this /historical folder for?

Reply:
@sandrina-p it makes it easier to remember potentially useful files that have been removed without having to go digging through the massive git history

Reply:
Hum.. oki